source/MapPlanetCard.cpp

```diff
@@ -44,10 +44,29 @@ MapPlanetCard::MapPlanetCard(const StellarObject &object, unsigned number, bool
 	hasShipyard = planet->HasShipyard();
 	hasOutfitter = planet->HasOutfitter();
 
-	reputationLabel = !hasSpaceport ? "No Spaceport" :
-		GameData::GetPolitics().HasDominated(planet) ? "Dominated" :
-		planet->GetGovernment()->IsEnemy() ? "Hostile" :
-		planet->CanLand() ? "Friendly" : "Restricted";
+	if(!hasSpaceport)
+		reputationLabel = "No Spaceport";
+	else
+	{
+		switch(planet->GetFriendliness())
+		{
+			case Planet::Friendliness::FRIENDLY:
+				reputationLabel = "Friendly";
+				break;
+			case Planet::Friendliness::RESTRICTED:
+				reputationLabel = "Restricted";
+				break;
+			case Planet::Friendliness::HOSTILE:
+				reputationLabel = "Hostile";
+				break;
+			case Planet::Friendliness::DOMINATED:
+				reputationLabel = "Dominated";
+				break;
+			default:
+				reputationLabel = "Friendly";
+				break;
```

Comment:
```suggestion
			case Planet::Friendliness::RESTRICTED:
				reputationLabel = "Restricted";
				break;
			case Planet::Friendliness::HOSTILE:
				reputationLabel = "Hostile";
				break;
			case Planet::Friendliness::DOMINATED:
				reputationLabel = "Dominated";
				break;
			case Planet::Friendliness::FRIENDLY:
			default:
				reputationLabel = "Friendly";
				break;
```
you could just make it fall in like this

Reply:
```suggestion
			default:
			case Planet::Friendliness::FRIENDLY:
				reputationLabel = "Friendly";
				break;
			case Planet::Friendliness::RESTRICTED:
				reputationLabel = "Restricted";
				break;
			case Planet::Friendliness::HOSTILE:
				reputationLabel = "Hostile";
				break;
			case Planet::Friendliness::DOMINATED:
				reputationLabel = "Dominated";
				break;
```

Reply:
Idk if that works like that but mb it does, ~~just make sure not everything is counted as Friendly~~
I've always seen the default below, but not sure if its needed